gradio/components/dataframe.py

```diff
@@ -46,9 +42,9 @@ def _import_polars():
 
 
 class DataframeData(GradioModel):
-    headers: List[str]
-    data: Union[List[List[Any]], List[Tuple[Any, ...]]]
-    metadata: Optional[Dict[str, Optional[List[Any]]]] = None
+    headers: list[str]
```

Comment:
Can you use the builtin as a type in 3.10?

Reply:
Yes, actually you could do this even in earlier versions of python!